### ISSUE TITLE
give option to avoid reformatting code or at least respect prettier rc in project root

### ISSUE THREAD

**Issue by emahuni** [strapi/codemods#40]
There are coding conventions that teams follow that this mod is assuming as THE standard. However, it should be to the engineer what to transform. So give them the option to disable reformatting of code by both disabling prettier completely and/or using prettier .prettierrc.js file... in project root.

 I had to fork and comment out the line that formats the code.

**Comment by derrickmehaffy** [comment]
Can you be specific about which part you are referring to and what you had to comment out?

**Comment by emahuni** [comment]
So sorry for being so vague, I modified these 2 lines with as minimum impact as possible (though the some of the logic is no longer necessary when this done this way, I didn't want to break anything and then fix it): https://github.com/strapi/codemods/blob/89429aa99a4ebffad35196dc4dfb1dc52905ee61/lib/global/utils/format-code.js#L11 

so from this:

```js
const formatFile = async (path) => {
  try {
    const fileContent = await readFile(path, 'utf-8');
    // Format the code with prettier
    return writeFile(
      path, prettier.format(fileContent, {
         filepath: path,
       })
    );
  } catch (error) {
    logger.warn(`Failed to format code, check ${path}`);
  }
};
```

to this: 
```js
const formatFile = async (path) => {
  try {
    const fileContent = await readFile(path, 'utf-8');
    // Format the code with prettier
    return writeFile(
      path, fileContent
    );
  } catch (error) {
    logger.warn(`Failed to format code, check ${path}`);
  }
};
```

This was causing a lot of undesired results. We tried to use an `rc` file and everything and noticed that it's not respecting any of that.

**Comment by markkaylor** [comment]
Good point. I'm starting to think we should probably just remove the formatting and leave that up to the end user. I think the format function is only called in one spot so the clean up should be pretty quick.

**Comment by derrickmehaffy** [comment]
> Good point. I'm starting to think we should probably just remove the formatting and leave that up to the end user. I think the format function is only called in one spot so the clean up should be pretty quick.

@emahuni are you willing to submit a PR to remove the code formatting? If so I can review it and merge it in as I agree with Mark here that we should just get rid of it.

**Comment by emahuni** [comment]
I didn't remove the prittierrc file, not sure what it is used for.
